nautobot/dcim/forms.py

```diff
@@ -2210,7 +2217,7 @@ class Meta:
             "position",
             "face",
             "rack_group",
-            "cluster",
+            "clusters",
```

Comment:
Possibly remove this? I can't remember if we actually support nulling M2M fields via bulk-edit.